trio/tests/test_gen_exports.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+import ast
+
+import trio
+from trio import _core
+
+
+def test_docstring_is_found():
+    pass
```

Comment:
I'm actually not sure that there are a lot of tests to write here... we're testing the generated code pretty thoroughly by just running the regular Trio test suite.

I guess we could put in a test that the docstrings are passing through correctly, since that's something that doesn't affect normal functionality so the rest of the test suite wouldn't catch it. Could just be

```python
assert trio._core._run.Runner.current_time.__doc__ != ""
assert trio._core._run.Runner.current_time.__doc__ == trio.current_time.__doc__
```

Eventually we'll want to add tests that pylint/mypy can now trace references like `trio.current_time` back to find their signature, but that doesn't have to happen here.

The main test that's missing is: we should actually call `gen_exports.py --test` inside `check.sh`, so that we'll get alerted if someone submits a PR where the generated file is out-of-sync :-)

Reply:
Added the test to `check.sh`. Doc string test not yet done, the function was more or less a prototype for the function inside `gen_exports.py`.